app/src/main/java/ar/scacchipa/twittercloneapp/presentation/LoginCloneActivity.kt

```diff
@@ -1,4 +1,4 @@
-package ar.scacchipa.twittercloneapp.component
+package ar.scacchipa.twittercloneapp.presentation
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
```

Comment:
What is this?? if is a login related, should be inside the login folder, also the **name** why Clone? what are you cloning ???

Reply:
TCA-55 The word 'Clone' was removed from the Activity identifiers